vnext/ReactWindowsCore/ShadowNode.h

```diff
@@ -33,6 +33,7 @@ struct ShadowNode {
   int64_t m_parent = -1;
   IViewManager *m_viewManager = nullptr;
   bool m_zombie = false;
+  bool m_isWindowedPopup = false;
```

Comment:
>m_isWindowedPopup [](start = 7, length = 17)

The popup can be windowed and non-windowed in xaml based on the OS version.
So WindowedPopup has special meaning. I guess you want to say: m_isPopupOpened?

Reply:
Well, the intent is to stop the traversal when you encounter a node that is a XAML window, but I'm moving this check up into NativeUIManager, so I'll remove this property.

---
In reply to: [304628094](https://github.com/microsoft/react-native-windows/pull/2781#discussion_r304628094) [](ancestors = 304628094)